### PR TITLE
Rtrt email interactives event

### PR DESCRIPTION
**Pull request by gtrufitt**
@sammorrisdesign This is the counterpart to the mods I've made on the interactive (which I can't push right now...).

### CODE REVIEW

File: static/src/stylesheets/module/email/_header.scss [hunk 1]
```diff
@@ -62,3 +62,15 @@
     }
 }
 
+// PLAIN MODS
```
Comment (johnduffell) [inline static/src/stylesheets/module/email/_header.scss]: what's a plain mod?

Comment (gtrufitt) [inline static/src/stylesheets/module/email/_header.scss]: The plain version of the component style modification (Ie: BEM modifier). Comment probably makes more sense in context when you can see there are a different modifications: https://github.com/guardian/frontend/blob/RTRT_Email-interactives-event/static/src/stylesheets/module/email/_header.scss

**Comment by johnduffell** [comment]
:+1:

File: static/src/stylesheets/module/email/_header.scss (hunk 1 above)
Comment (johnduffell) [inline static/src/stylesheets/module/email/_header.scss]: I'm still confused but I'm not 100% up on BEM terminology, maybe run it by someone else to confirm it makes sense!

**Pull request merged**